charts/spire/charts/spire-server/values.yaml

```diff
@@ -104,6 +104,16 @@ upstreamAuthority:
         certificate: ""
         key: ""
         bundle: ""
+  certManager:
+    enabled: false
+    rbac:
+      create: true
+    issuer_name: "spire-ca"
+    issuer_kind: "Issuer"
+    issuer_group: "cert-manager.io"
+    # -- Specify to use a namespace other then the one the chart is installed into
+    namespace: ""
```

Comment:
Is this the namespace certmanager is installed into? Should we default this to `cert-manager`, I think that is where all cert-manager tutorials also put it.

Reply:
no. its the namespace to register the cert requests into. if not set, it does so in the same namespace as spire. which is right most of the time. It was in the docs as an option though so I added it too.